dataset_test.go

```diff
@@ -77,6 +89,7 @@ func TestDataSetByType(t *testing.T) {
 		Adfraud:        &DataSet_AdFraud{},
 		Malicious:      &DataSet_Malicious{},
 		Echo:           &DataSet_Echo{},
+		Keyword:        make(map[string]*DataSet_Sentiment),
```

Comment:
Keyword:   &DataSet_Keyword
and
Sentiment: &DataSet_Sentiment 
to be consistent with the other DataSet types.